infra/main.parameters.json

```diff
@@ -55,6 +55,9 @@
     },
     "gptDeploymentName": {
       "value": "${AZURE_OPENAI_GPT_DEPLOYMENT=davinci}"
+    },
+    "useApplicationInsights": {
+      "value": "${AZURE_USE_APPLICATION_INSIGHTS=false}"
```

Comment:
I notice the TODO templates also output the similar USE_APIM in main.bicep - am not sure when/if thats needed for this scenario: https://github.com/pamelafox/azure-dev/blob/37f88d5196a3aed4899f2928a67f43714dbff967/templates/todo/projects/csharp-sql/.repo/bicep/infra/main.bicep#L200

If you're not sure either, maybe @vhvb1989 can chime in.